src/amo/components/ErrorPage/NotFound/index.js

```diff
@@ -26,33 +26,41 @@ export class NotFoundBase extends React.Component<InternalProps> {
   render() {
     const { i18n } = this.props;
 
-    // We use `getLocalizedTextWithLinkParts()` two times to create all the
-    // variables needed to display both the `Link` components and texts
-    // before/after.
-    let linkParts = getLocalizedTextWithLinkParts({
```

Comment:
Can you file an issue to delete `getLocalizedTextWithLinkParts()` and replace all calls? There is no reason for the old function anymore. I don't want someone to accidentally use it by copying/pasting code.

Reply:
Yep, #7866.